app/controllers/concerns/development_dependency_checks.rb

```diff
@@ -12,7 +12,6 @@ module DevelopmentDependencyChecks
   def verify_sidekiq_running
     return if Sidekiq::ProcessSet.new.size.positive?
 
-    flash[:global_notice] = "Sidekiq is not running and is needed for the app to function properly. \
-                           Use bin/startup to start the application properly.".html_safe
+    flash[:global_notice] = I18n.t("concerns.development_dependency_checks.sidekiq_is_not_running_and")
```

Comment:
This is not directly user-facing as it's only enabled in local development. Given we generally require people to reasonably comfortable in English to contribute I'm not sure there's much value in translating this.

Reply:
> This is not directly user-facing as it's only enabled in local development.

I see. Thank you for the input.